components/tooltip/README.md

```diff
@@ -0,0 +1,66 @@
+# Tooltips
+## d2l-tooltip
+
+The `d2l-tooltip` component is used to display additional information when users focus or hover on a point of interest.
+
+![screenshot of an error tooltip](./screenshots/tooltip-error.png)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/tooltip/tooltip.js';
+</script>
+
+<input type="text" placeholder="Hover for Error" id="tooltip-error">
+<d2l-tooltip for="tooltip-error" state="error" align="start" offset="10">
+	Your error message will display here
+</d2l-tooltip>
+```
```

Comment:
Can you add a note in here about how the the `for` and corresponding `id` element need to be in the same shadow root?